lib/question.js

```diff
@@ -1378,6 +1378,7 @@ module.exports = {
     if (!assessment) {
       // instructor question pages
       const questionUrl = urlPrefix + '/question/' + question.id + '/';
+      urls.questionUrl = questionUrl;
```

Comment:
I think we also need this in the `else` branch of the `if (!assessment)` statement so that student questions will also have this defined. At the moment we get this on student pages:
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/875833/216115905-1fffbd63-a6c5-4791-af47-84876d1318fb.png">

Reply:
Good call! Will add.

Reply:
Fixed in af78b02.